docs/contributor/CONTRIBUTING.md

```diff
@@ -82,10 +82,7 @@ The reviewers are also responsible to check:
 All Pull Requests must contain proper title & description, as described in [Pull Request
 Template](./PULL_REQUEST_TEMPLATE.md). Moreover, all pull requests must have a proper `prdoc` file attached.
 
-Some Pull Requests can be exempt of `prdoc` documentation, those must be labelled with
-[`R0-silent`](https://github.com/paritytech/labels/blob/main/ruled_labels/specs_polkadot-sdk.yaml#L95-L97).
-
-Non "silent" PRs must come with documentation in the form of a `.prdoc` file.
+Pull Requests labelled with ⁠`R0-no-crate-publish-required` are exempt from ⁠prdoc documentation requirements.
```

Comment:
Are we actually using https://github.com/paritytech/labels in a meaningful way? if yes, we should update the name "Silent" there as well. If not, we should archive it.

Reply:
I will update it there yes, but that repo is not that useful in general IMO.